priv/repo/migrations/20250811220833_add_timezone_to_sections.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Oli.Repo.Migrations.AddTimezoneToSections do
+  use Ecto.Migration
+
+  def change do
+    alter table(:sections) do
+      add :timezone, :string, default: "Etc/UTC"
```

Comment:
Perhaps the default here should be nil, so that the fallback logic works as intended? Otherwise there is no good way to know whether someone explicitly set this timezone for a section or if it was just the default. I could imagine a feature where some torus deployments may want to set a system-level default timezone